build.gradle

```diff
@@ -372,9 +385,38 @@ task createApiDocsRedirect {
   }
 }
 
+task publishDocs {
+  dependsOn 'closeAndReleaseRepository'
+  doLast {
```

Comment:
@garrettjonesgoogle The current process clones a clean gh-pages before each release and removes the dir after the release is done.